src/runtime_src/core/include/xrt/xrt_graph.h

```diff
@@ -28,6 +28,7 @@
 # include <chrono>
 # include <string>
 # include <cstdint>
+# include "experimental/xrt_hw_context.h"
```

Comment:
Move to xrt group of includes and sort within group.
If xrt/xrt_hw_context.h does not compile for C, then fix this header, please.
```suggestion
# include "xrt/xrt_hw_context.h"
```